docker-compose.yaml

```diff
@@ -0,0 +1,22 @@
+version: '3.5'
+
+services:
+
+  telegram-bot:
+    build:
+      dockerfile: Dockerfile
+    container_name: "app"
+    ports:
+      - "8080:8080"
+    depends_on:
```

Comment:
mongodb:
        condition: service_healthy
        
A little advice on how to improve it